ci-operator/jobs/dora-metrics/pelorus/dora-metrics-pelorus-master-presubmits.yaml

```diff
@@ -1,251 +1,5 @@
-          secretName: registry-pull-credentials
-      - name: result-aggregator
-        secret:
-          secretName: result-aggregator
-    trigger: (?m)^/test( | .* )4.11-e2e-openshift,?($|\s.*)
-  - agent: kubernetes
-    always_run: true
-    branches:
-    - ^master$
-    - ^master-
-    cluster: build03
-    context: ci/prow/4.12-e2e-openshift
-    decorate: true
-    decoration_config:
-      skip_cloning: true
-    labels:
-      ci-operator.openshift.io/cloud: aws
-      ci-operator.openshift.io/cloud-cluster-profile: aws
-      ci-operator.openshift.io/variant: "4.12"
-      ci.openshift.io/generator: prowgen
-      pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-dora-metrics-pelorus-master-4.12-e2e-openshift
-    rerun_command: /test 4.12-e2e-openshift
-    spec:
-      containers:
-      - args:
-        - --gcs-upload-secret=/secrets/gcs/service-account.json
-        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
-        - --lease-server-credentials-file=/etc/boskos/credentials
-        - --report-credentials-file=/etc/report/credentials
-        - --secret-dir=/secrets/ci-pull-credentials
-        - --secret-dir=/usr/local/e2e-openshift-cluster-profile
-        - --target=e2e-openshift
-        - --variant=4.12
-        command:
-        - ci-operator
-        image: ci-operator:latest
-        imagePullPolicy: Always
-        name: ""
-        resources:
-          requests:
-            cpu: 10m
-        volumeMounts:
-        - mountPath: /etc/boskos
-          name: boskos
-          readOnly: true
-        - mountPath: /secrets/ci-pull-credentials
-          name: ci-pull-credentials
-          readOnly: true
-        - mountPath: /usr/local/e2e-openshift-cluster-profile
-          name: cluster-profile
-        - mountPath: /secrets/gcs
-          name: gcs-credentials
-          readOnly: true
-        - mountPath: /secrets/manifest-tool
-          name: manifest-tool-local-pusher
-          readOnly: true
-        - mountPath: /etc/pull-secret
-          name: pull-secret
-          readOnly: true
-        - mountPath: /etc/report
-          name: result-aggregator
-          readOnly: true
-      serviceAccountName: ci-operator
-      volumes:
-      - name: boskos
-        secret:
-          items:
-          - key: credentials
-            path: credentials
-          secretName: boskos-credentials
-      - name: ci-pull-credentials
-        secret:
-          secretName: ci-pull-credentials
-      - name: cluster-profile
-        secret:
-          secretName: cluster-secrets-aws
-      - name: manifest-tool-local-pusher
-        secret:
-          secretName: manifest-tool-local-pusher
-      - name: pull-secret
-        secret:
-          secretName: registry-pull-credentials
-      - name: result-aggregator
-        secret:
-          secretName: result-aggregator
-    trigger: (?m)^/test( | .* )4.12-e2e-openshift,?($|\s.*)
   - agent: kubernetes
     always_run: true
```

Comment:
Lets address this issue here https://github.com/dora-metrics/pelorus/issues/1076

we need to change here to false https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/oadp-operator/openshift-oadp-operator-master-presubmits.yaml#L979

and add this `run_if_changed` field https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/oadp-operator/openshift-oadp-operator-master-presubmits.yaml#L996

I think this regex should be enough `"^exporters/|^charts/|^pelorus-operator/"`

Reply:
what about Makefile and files that are in fact changing e2e tests?

Reply:
```^exporters/|^charts/|^pelorus-operator/|^scripts|Makefile```

?

Reply:
Good though, I agree

Reply:
wait, actually not

Reply:
If we add Makefile and script folder, we will still get some false positives

I suggest adding just `scripts/run-pelorus-e2e-tests.sh` file, which we use for running E2E tests

so, I would suggest something like `^exporters/|^charts/|^pelorus-operator/|^scripts/run-pelorus-e2e-tests.sh$`